source/features/clear-pr-merge-commit-message.tsx

```diff
@@ -49,10 +49,6 @@ void features.add(import.meta.url, {
 		// Don't clear 1-commit PRs #3140
 		() => select.all('.TimelineItem.js-commit').length === 1,
 	],
-	additionalListeners: [
-		onPrMergePanelOpen,
```

Comment:
@RussKie I'm changing which event I use to change the message, maybe that would fix the race condition

- https://github.com/refined-github/refined-github/issues/6348

Reply:
On this very PR, the feature didn't even clear the field (using the old listener), so this new observer should actually fix at least one bug